### ISSUE TITLE
sensor.garo_wallbox_status getting UNAVAILABLE instead of CHARGING_PAUSED

### ISSUE THREAD

**Issue by peternorlander** [sockless-coding/garo_wallbox#61]
Hi and thanks for an awesome integration!

I updated to the latest version 2024.10.0, (think I was using v1.0.11 before this) and have noticed a strange behaviour of the sensor.garo_wallbox_status entity. When sensor.garo_wallbox = ALWAYS_OFF and I'm connecting my car its not getting the expected CHARGING_PAUSED but instead UNAVAILABLE. But if I view the entity its "friendly name" is correct but not the code.

See this printscreen:
![image](https://github.com/user-attachments/assets/aac2ad25-d92c-45ea-80e8-8db868bf1d08)

I'm running
Garo firmware: 1.3.7
HA Core: 2024.10.4
HA Supervisor: 2024.10.3
HA Operating System: 13.2

**Comment by peternorlander** [comment]
Is nobody else is facing this issue? For me I really miss the old behaviour because all nice automations I built to detect when my car becomes connected isn't working anymore. Triggering based on the status sets to "UNAVAILABLE" isn't as reliable. ;) Please let me know if you need something to figure out the problem.

**Comment by eclaug** [comment]
Yes, i recently noticed that I also have this error. The states of the entity sensor.garo_status in the History view is more pretty like "Charging", "Disconnected" etc. But if I go to developer tools and list the current state {{ states("sensor.garo_status") }} I get other values. This make my automations a lot harder to configure when triggering on the states.
